cmake/scripts/rbpi/ArchSetup.cmake

```diff
@@ -13,9 +13,12 @@ else()
   if(CPU STREQUAL arm1176jzf-s)
     set(ARCH arm)
     set(NEON False)
+    set(NEON_FLAGS "-mcpu=arm1176jzf-s -mtune=arm1176jzf-s -mfloat-abi=hard -mfpu=vfp")
   elseif(CPU MATCHES "cortex-a7" OR CPU MATCHES "cortex-a53")
     set(ARCH arm)
     set(NEON True)
+    set(NEON_FLAGS "-fPIC -mcpu=cortex-a7 -mfloat-abi=hard -mfpu=neon-vfpv4 -mvectorize-with-neon-quad")
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lpthread")
```

Comment:
then those linker flags are a relic workaround for missing checks somewhere in our depends.
We build perfectly fine without that, so please remove that line.